packages/kolibri-tools/lib/i18n/utils.py

```diff
@@ -6,66 +6,25 @@
 import os
 import sys
 
-try:
-    import kolibri_exercise_perseus_plugin
-
-    PERSEUS_LOCALE_PATH = os.path.join(
-        os.path.dirname(kolibri_exercise_perseus_plugin.__file__), "locale"
-    )
-    PERSEUS_SOURCE_PATH = os.path.join(PERSEUS_LOCALE_PATH, "en", "LC_MESSAGES")
-except ModuleNotFoundError:
-    # We don't throw an error here as it will be handled later if needed.
-    # Not all projects depend on our Perseus plugin
-    PERSEUS_LOCALE_PATH = None
-    PERSEUS_SOURCE_PATH = None
-
-# Let's make sure the paths actually exist
-if PERSEUS_SOURCE_PATH and not os.path.exists(PERSEUS_SOURCE_PATH):
-    os.makedirs(PERSEUS_SOURCE_PATH)
-if PERSEUS_LOCALE_PATH and not os.path.exists(PERSEUS_LOCALE_PATH):
-    os.makedirs(PERSEUS_LOCALE_PATH)
+import configparser
 
 logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.INFO)
 logging.StreamHandler(sys.stdout)
 
 
-PROJECT_NAME = os.getenv("CROWDIN_PROJECT", "kolibri")
-
-THIS_FILE_PATH = os.path.abspath(os.getcwd())
-
-LOCALE_OPTIONS = {
-    "kolibri": os.path.join(THIS_FILE_PATH, "kolibri/locale"),
-    "kolibri-studio": os.path.join(THIS_FILE_PATH, "contentcuration/locale"),
-}
-
-LOCALE_PATH = LOCALE_OPTIONS[PROJECT_NAME]
-
-if not LOCALE_PATH:
-    logging.error(
-        "\nCould not get LOCALE_PATH for CROWDIN_PROJECT {}".format(PROJECT_NAME)
-    )
-    sys.exit(1)
-
-try:
-    if not os.path.exists(LOCALE_PATH):
-        os.makedirs(LOCALE_PATH)
-except NotADirectoryError:
-    # This means you're not using the correct CROWDIN_PROJECT
-    logging.error(
-        "Please ensure that CROWDIN_PROJECT {} is correct and that you're running this command from the\
-                  root of that project. This failed trying to make this directory: {}".format(
-            PROJECT_NAME, LOCALE_PATH
-        )
-    )
-    sys.exit(1)
-
-SOURCE_PATH = os.path.join(LOCALE_PATH, "en", "LC_MESSAGES")
-
-if not os.path.exists(SOURCE_PATH):
-    os.makedirs(SOURCE_PATH)
+# Path to the kolibri locale language_info file, which we use if we are running
+# from inside the Kolibri repository.
+_KOLIBRI_LANGUAGE_INFO_PATH = os.path.join(
+    os.path.dirname(__file__), "../../../../kolibri/locale/language_info.json"
+)
```

Comment:
somewhat brittle coupling to details of existing repo structure. Perhaps should be pulled from `setup.cfg` or some other global per-project config?

Reply:
I did think about adding it there - mostly this is a workaround for us only keeping one copy of `language_info.json` in the repository (previously it was duplicated in the locale folder and in this folder), this allows us to only replicate this default file at build time of the package.

The reason I decided not to add it as a global configuration was because it should only need to be specified when our language_info.json is insufficient.